themes/luci-theme-material/luasrc/view/themes/material/header.htm

```diff
@@ -200,8 +210,79 @@
 	<script src="<%=url('admin/translations', luci.i18n.context.lang)%><%# ?v=PKG_VERSION %>"></script>
 	<script src="<%=resource%>/cbi.js"></script>
 	<script src="<%=resource%>/xhr.js"></script>
+	<script type="text/javascript">//<![CDATA[
+		document.addEventListener('DOMContentLoaded', function() {
+			document.querySelectorAll(".main > .loading").forEach(function (e) {
+				if (e) {
+					e.style.display = "none";
+				}
+			});
+
+			document.querySelectorAll(".main > .main-left > .nav > .slide > .menu").forEach(
+			function (e){
+				if(e){
+					e.addEventListener('click', function(){
+						var ul2 = this.nextElementSibling;
+						var tag = this;
+						if (ul2.style.display === "" || ul2.style.display === "none") {
+							ul2.classList.add("active");
+							tag.classList.add("active");
+							ul2.style.display = "block";
+						}else{
+							ul2.classList.remove("active");
+							tag.classList.remove("active");
+							ul2.style.display = "none";
+						}
+					});
+
+					e.nextElementSibling.querySelectorAll("a").forEach(
+					function (a){
+						if(a)
+						{
+							a.parentElement.addEventListener('click', function(ev){
+								ev.stopPropagation();
+								a.click();
+							});
+
+							a.addEventListener('click', function(ev){
+								ev.stopPropagation();
+							});
+						}
+					});
+
+					if (e.classList.contains("active")) {
+						e.click();
+					}
+				}
+			});
+
+			if (!document.querySelector('body').classList.contains('logged-in')) {
+				document.querySelector("body").classList.add("node-main-login");
+			};
+
+			var showSide = false;
+			document.querySelector(".showSide").addEventListener('click', function () {
+				if (showSide){
+					document.querySelector(".main-left").setAttribute("style", "width: 0");
+					document.querySelector(".main-right").setAttribute("style", "overflow-y: auto");
```

Comment:
`overflow-y` should be `visible` here, for mobile browsers to properly handle the scrollbar